infra/cifuzz/platform_config/prow.py

```diff
@@ -12,11 +12,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Module for getting the configuration CIFuzz needs to run on prow."""
+import logging
+import os
 import platform_config
 
 # pylint: disable=too-few-public-methods
 
 
 class PlatformConfig(platform_config.BasePlatformConfig):
   """CI environment for Prow."""
-  # TODO(mpherman2) Fill this out.
+
+  @property
+  def project_src_path(self):
+    """Returns the manually checked out path of the project's source if
+    specified or the current directory if not. Prow will run ClusterfuzzLite
+    at the directory head for the repo."""
+    project_src_path = os.getenv('PROJECT_SRC_PATH', '.')
+    logging.debug('PROJECT_SRC_PATH: %s.', project_src_path)
+    return project_src_path
+
+  @property
+  def workspace(self):
+    """Returns the workspace."""
+    # Let Prow user override workspace, but default to using artifacts dir
+    return os.getenv('WORKSPACE', os.getenv('ARTIFACTS'))
+
+  @property
+  def base_ref(self):
+    """Returns the base branch to diff against (pr fuzzing)."""
+    return os.getenv('PULL_BASE_REF')
+
+  @property
+  def project_repo_name(self):
+    """Returns the project repo name."""
+    return os.getenv('REPO_NAME')
+
+  @property
+  def base_commit(self):
+    """Returns the base commit to diff against (commit fuzzing)."""
+    return os.getenv('PULL_BASE_SHA')
```

Comment:
This env var is only used for commit fuzzing. Some projects like systemd do this where CIFuzz runs everytime main is commited to and then the diffing happens against the base_commit.
Is this value going to be correct when there is no pull request?

Reply:
For Periodic jobs this env var will not be set. It is only set for pre/post submits. See: https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#job-environment-variables

Reply:
Right but fuzzing on every commit to main/master isn't really a periodic job. It's in response to an event (like pull request fuzzing).
I can also explain this over video.
At the end of the day this doesn't really a ton, we don't document commit fuzzing in the docs anyway, but I'd prefer we got it right.

Reply:
From the docs it looks like Im talking about a postsubmit job? and that you did this correctly!